.github/ISSUE_TEMPLATE/2-feature-request.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Feature Request
+about: Use this template for raising a feature request
+---
+
+## System information
```

Comment:
Need the priority and impact information.

Reply:
Priority should be defined by internal scrum